hoomd/md/OPLSDihedralForceCompute.cc

```diff
@@ -92,10 +92,10 @@ pybind11::dict OPLSDihedralForceCompute::getParams(std::string type)
     ArrayHandle<Scalar4> h_params(m_params, access_location::host, access_mode::read);
     auto val = h_params.data[typ];
     pybind11::dict params;
```

Comment:
```suggestion
    pybind11::dict params;
    // note: the values stored in params are precomputed k/2 values
```
This * 2 is correct, but the class documentation doesn't make it very clear how this is stored. Add this as a reminder.